src/overlays/actors/ovl_Bg_Hidan_Dalm/z_bg_hidan_dalm.c

```diff
@@ -8,8 +8,16 @@ void BgHidanDalm_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Draw(Actor* thisx, GlobalContext* globalCtx);
+#define BgHidanDalm_WaitInteraction func_80885F4C
+void BgHidanDalm_WaitInteraction(BgHidanDalm* this, GlobalContext* globalCtx);
```

Comment:
we like to keep a line break between the main functions (that take thisx) and the other ones.
also you can rename the function in the init asm file to get around needing the define